cmd/node/main.go

```diff
@@ -1383,47 +1380,6 @@ func closeAllComponents(
 	chanCloseComponents <- struct{}{}
 }
 
-func handleAppClose(log logger.Logger, endProcessArgument endProcess.ArgEndProcess) {
-	log.Debug("closing node")
-
-	switch endProcessArgument.Reason {
-	case core.ShuffledOut:
```

Comment:
leave closing node final log

Reply:
done. also added `description` to the internal signal close log